apps/studio/src/server/modules/user/__tests__/user.service.test.ts

```diff
@@ -51,20 +53,29 @@ describe("user.service", () => {
   describe("createUserWithPermission", () => {
     const TEST_EMAIL = "test@open.gov.sg"
     let siteId: number
+    let creatorUserId: string
 
     beforeAll(async () => {
       await setUpWhitelist({ email: TEST_EMAIL })
     })
 
     beforeEach(async () => {
-      await resetTables("User", "ResourcePermission", "Site")
+      await resetTables("User", "ResourcePermission", "Site", "AuditLog")
       const { site } = await setupSite()
       siteId = site.id
+
+      const creator = await setupUser({
+        name: "creator",
+        email: "creator@open.gov.sg",
+        isDeleted: false,
+      })
+      creatorUserId = creator.id
```

Comment:
nit - i think more useful next time to just have `creator` then assign in the `beforeEach` rather than having a `creatorUserId`

Reply:
oh my thought process is that since `creator` is not being used beyond its `creatorUserId`, and that `creatorUserId` sole purpose is to pass it into the auditlog, there's no need to expose the `creator` to the tests